spacy/lang/sl/punctuation.py

```diff
@@ -0,0 +1,64 @@
+from ..char_classes import LIST_ELLIPSES, LIST_ICONS, HYPHENS, LIST_PUNCT, LIST_QUOTES, CURRENCY, UNITS, PUNCT, \
+    LIST_CURRENCY, CONCAT_QUOTES
+from ..char_classes import CONCAT_QUOTES, ALPHA_LOWER, ALPHA_UPPER, ALPHA
+from ..char_classes import merge_chars
+from ..punctuation import TOKENIZER_PREFIXES as BASE_TOKENIZER_PREFIXES
+
+_prefixes = ["•", "\-", "\/", "'"] + BASE_TOKENIZER_PREFIXES
+
+_suffixes = (
+        ["—", "–", "\-", "\:", "\+", "\/", "=", "×", "\$", "•", "¯", "%"]
```

Comment:
These should either be double-escaped (e.g. `"\\+"` rather than `"\+"`) or the strings should be preceded with `r`, e.g. `r"\+"`

Reply:
The lists have been cleaned and duplicates removed.